site/content/pages/organizing/04-Running your event/07-video-streaming.txt

```diff
@@ -0,0 +1,17 @@
+---
+extension: html
+filter:
+- erb
+- markdown
+title: Video - recording and streaming
+---
+
+Whether or not you're able to livestream, it's important to record all the talks. This is invaluable for your speakers, and it's great for the community. Your audiovisual company should be able to provide a camera or cameras and record the feed off the board into a computer.
```

Comment:
@bridgetkromhout I wonder if we should even say "required" or whether that's too strong? We definitely want video, but maybe saying it's important, and precedence that it's always been done means we don't have to use stronger language... not sure?